src/test/java/uk/gov/hmcts/reform/logging/filters/RequestStatusLoggingFilterTest.java

```diff
@@ -49,36 +51,40 @@ public void removeAppender() {
     @Test
     public void logsSuccessfulRequest() throws IOException, ServletException {
         new RequestStatusLoggingFilter(FROZEN_CLOCK).doFilter(
-                requestWithMethodAndUri("GET", "/some/path"),
+                requestWithMethodAndUri(GET, SOME_PATH),
                 responseWithStatus(400),
                 mock(FilterChain.class)
         );
 
-        Map<String, Object> fields = new HashMap<>();
-        fields.put("requestMethod", "GET");
-        fields.put("requestUri", "/some/path");
+        Map<String, Object> fields = new ConcurrentHashMap<>();
+        fields.put("requestMethod", GET);
+        fields.put("requestUri", SOME_PATH);
         fields.put("responseTime", 0L);
         fields.put("responseCode", 400);
 
-        testAppender.assertEvent(0, INFO, "Request GET /some/path processed in 0ms", appendEntries(fields));
+        String message = "Request " + GET + " " + SOME_PATH + " processed in 0ms";
+
+        testAppender.assertEvent(0, INFO, message, appendEntries(fields));
     }
 
     @Test
     public void logsFailedRequest() throws IOException, ServletException {
         thrown.expect(RuntimeException.class);
 
         new RequestStatusLoggingFilter(FROZEN_CLOCK).doFilter(
-                requestWithMethodAndUri("GET", "/some/path"),
+                requestWithMethodAndUri("GET", SOME_PATH),
                 responseWithStatus(-1),
                 failingFilterChain()
         );
 
-        Map<String, Object> fields = new HashMap<>();
+        Map<String, Object> fields = new ConcurrentHashMap<>();
         fields.put("requestMethod", "GET");
-        fields.put("requestUri", "/some/path");
+        fields.put("requestUri", SOME_PATH);
         fields.put("responseTime", 0L);
 
-        testAppender.assertEvent(0, ERROR, "Request GET /some/path failed in 0ms", appendEntries(fields));
+        String message = "Request " + GET + " " + SOME_PATH + " failed in 0ms";
+
+        testAppender.assertEvent(0, ERROR, message, appendEntries(fields));
     }
 
     private HttpServletRequest requestWithMethodAndUri(String method, String url) {
```

Comment:
not really related to changes in this pull request, but is this method every called with something other than `"GET"`?